docs/theming/_index.md

```diff
@@ -259,6 +259,26 @@ Spacing variables get prepended with `--oc-space-`, so e.g. _"xlarge"_ creates t
 }
 ```
 
+### App Banner
+To configure the app banner shown on mobile devices, you can use the following settings:
+
+```json 
+{
+  "appBanner": {
+    "title": "ownCloud",
+    "publisher": "ownCloud GmbH",
+    "additionalInformation": "",
+    "ctaText": "OPEN",
+    "icon": "themes/owncloud/assets/owncloud-app-icon.png",
+    "appScheme": "owncloud"
+  }
+}
```

Comment:
Could you please add this block to the example theme down below as well? To give the user a hint where in the config it needs to be located.